GET operation on /log-level

FedericoMassaioli opened the pull request:
Hi,
I added a GET operation to /log-level to recover available contextNames and currently set log levels.

Cheers
Federico

Eomm commented on plugin.js:
```diff
@@ -34,6 +34,26 @@ async function fastifyLogController (fastify, opts) {
 
   const logLevels = Object.keys(fastify.log.levels.values)
 
+  fastify.get('/log-level', {
```
To avoid a breaking change, I would hide this route behind an option:

```
app.register(require('fastify-log-controller'), {
  exposeGet: true // default: false
})
```

FedericoMassaioli replied:
Sure!
I'll take care of it in the next few days.

Eomm reviewed (commented):
Thanks for the PR 🚀

All good, I would change a bit how we add the route!

FedericoMassaioli commented:
Added option to control GET routes exposure.
While I was at it, I also added a route to get defined log levels. I could have folded the functionality in the original route, but I prefer simpler endpoints, they are easier to cope with when you want to set up management consoles/tools. I can change it if you prefer the combined approach.

Eomm reviewed (approved):
Thanks for the feature

Exposing a decorator to access the same info could give to the user the ability to create the get route by itself.
But I think a decorator is heavier than a route in this case and for sure more straightforward.

thanks for the PR!
Landing!

FedericoMassaioli commented:
Thanks to you for the useful package! :-)
